src/Compilers/VisualBasic/Test/IOperation/IOperation/IOperationTests_IUsingStatement.vb

```diff
@@ -1008,7 +1008,7 @@ IExpressionStatementOperation (OperationKind.ExpressionStatement, Type: null) (S
 
         <CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)>
         <Fact()>
-        Public Sub UsingFlow_14()
+        Public Sub UsingFlow_01()
```

Comment:
>UsingFlow_01 [](start = 19, length = 12)

Why was that renamed? Is it possible that it corresponded to C# test with the same name?

Reply:
Long ago there were `UsingFlow_{1-13}` in C# and then {`14,15}` in VB, but C# has since added significantly more tests with `{14...}` so it makes no sense to have a random set of numbered VB tests. We can move these to the end of the C# tests, but the likelihood is just that they'll clash again.

Reply:
Ideally we'd give them descriptive names tbh

Reply:
> We can move these to the end of the C# tests, but the likelihood is just that they'll clash again. 

There is no requirement for test methods to be sorted in any way. The test was not affected by the change as well. I find the rename unnecessary.

---
In reply to: [533652564](https://github.com/dotnet/roslyn/pull/49481#discussion_r533652564) [](ancestors = 533652564)